Bottom: 076c5fb82a837007146dc1677f907a11148793e3
Top:    c322f0b5b4452cbfdf49b026c85b0e2738ed1e62
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-27 02:32:02 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
